src/views/SpaceProposal.vue

```diff
@@ -120,6 +136,18 @@ async function loadResults() {
   }
 }
 
+const { loadBy, loadingMore, stopLoadingMore, loadMore } =
+  useInfiniteLoader(10);
+
+async function loadMoreVotes() {
+  const votesObj = await getProposalVotes(id, {
+    first: loadBy,
+    skip: votes.value.length
+  });
+  stopLoadingMore.value = votesObj?.length < loadBy;
```

Comment:
This will show the "see more" button, even if all are loaded when the total number is a multiple of 10. We will need some form of counter in graphql I guess.

Reply:
Yes, we should open an issue for this.

Reply:
I added this as a comment in the new issue you opened. Can be implemented for votes only first.

Reply:
Actually this could already be done with `voteCount`. Will change that before I merge